### ISSUE TITLE
Determine optimal way to utilize helper alongside multiple versions of Core

### ISSUE THREAD

**Issue by wbnns** [cornelius/rpc-docs-helper#3]
It would be nice to set a foundation for expanding the helper in a way that it will adhere to a model of compatibly generating documentation for multiple versions of Core on Bitcoin.org.

There would of course need to be frontend changes to the site to allow the user to more efficiently browse through documentation for different versions, but just wanted to mention the goal here as well to stay in sync if we decide to head in that direction.

**Comment by cornelius** [comment]
@wbnns How do you envision the presentation of multiple versions? Something like a global switch to select for which version you want to see the documentation? Or something per command? Or even showing the information for multiple versions in the same document (with notes what applies to which version)?

A global switch would probably be the simples thing to do, and it would also be in line with how others such as the Python docs, or the docs on ReadTheDocs are doing it.

**Comment by wbnns** [comment]
@cornelius 

> A global switch would probably be the simples thing to do, and it would also be in line with how others such as the Python docs, or the docs on ReadTheDocs are doing it.

Agreed. Was thinking the same thing that perhaps it would be ideal to have a global switch for each version.
